google-compute-engine/src/test/java/org/jclouds/googlecomputeengine/internal/BaseGoogleComputeEngineApiLiveTest.java

```diff
@@ -73,7 +76,9 @@ public BaseGoogleComputeEngineApiLiveTest() {
    }
 
    @Override protected GoogleComputeEngineApi create(Properties props, Iterable<Module> modules) {
-      Injector injector = newBuilder().modules(modules).overrides(props).buildInjector();
+      // BouncyCastleCrypto required for ResetWindowsPassword related tests
+      injector = newBuilder().modules(ImmutableSet.<Module>builder().addAll(modules).add(new BouncyCastleCryptoModule()).build())
```

Comment:
What about providing a protected `modules` method that subclasses can override to contribute modules? This way each subclass would be able to add custom modules.